src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.Trivia.cs

```diff
@@ -28,9 +26,7 @@ public void it_can_parse_a_string_with_only_whitespace()
 
             result.SyntaxTree.RootNode
                   .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
-                  .MethodNode.ChildTokens.First().TextWithTrivia.Should().Be(" \t ");
-
-            // TODO: Test error reporting.
+                  .MethodNode.ChildTokens.First().Text.Should().Be(" \t ");
```

Comment:
In this case I think we'll need to report an error for missing URL. This is probably a common state that the parser will find itself in as the user edits a file / cell.

Reply:
We can go deeper into error reporting in a future PR.